manifests/example-template-windows.yml

```diff
@@ -0,0 +1,67 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: windows-vm-template
+  annotations:
+    description: "OCP kubevirt windows VM template"
+    tags: "kubevirt,ocp,template,windows"
+  labels:
+    miq.github.io/kubvirt/os: windows-2016
+    miq.github.io/kubevirt-is-vm-template: true
+objects:
+- apiVersion: kubevirt.io/v1alpha1
+  kind: OfflineVirtualMachine
+  metadata:
+    name: ${NAME}
+  spec:
+    template:
+      spec:
+        domain:
+          features:
+            acpi: {}
+            apic: {}
+            hyperv:
+              relaxed: {}
+              vapic: {}
+              spinlocks:
+                spinlocks: 8191
+          clock:
+            utc: {}
+            timer:
+              hpet:
+                present: false
+              pit:
+                tickPolicy: delay
+              rtc:
+                tickPolicy: catchup
+              hyperv: {}
+          cpu:
+            cores: ${CPU_CORES}
+          machine:
+            type: q35
+          resources:
+            requests:
+              memory: ${MEMORY}
+          devices:
+            disks:
+              - name: disk0
+                volumeName: disk0-pvc
+                disk:
```

Comment:
Please drop this and the next line.

I'd like to avoid the `dev` field.

Reply:
done